pkg/cmd/server/kubernetes/node_config.go

```diff
@@ -169,6 +171,32 @@ func BuildKubernetesNodeConfig(options configapi.NodeConfig) (*NodeConfig, error
 		return nil, err
 	}
 
+	// Replace the standard k8s emptyDir volume plugin with a wrapper version
+	// which offers XFS quota functionality, but only if the node config
+	// specifies an empty dir quota to apply to projects:
+	if options.ProjectEmptyDirQuota != nil {
+		glog.V(2).Info("Replacing empty-dir volume plugin with quota wrapper")
+		wrappedEmptyDirPlugin := false
+		for idx, plugin := range cfg.VolumePlugins {
+			// Can't really do type checking or use a constant here as they are not exported:
+			if plugin.Name() == "kubernetes.io/empty-dir" {
```

Comment:
@deads2k the binary process that starts up kubelet and the controllermanager is the part that knows what the volume plugin list is.  Currently, Origin looks to the upstream binary for the list, which is all plugins.  We can configure our own list. Suggestions on where this would go?